services/controllerhost/config.go

```diff
@@ -49,15 +49,22 @@ type (
 	OutputPlacementConfig struct {
 		AdminStatus string `name:"adminStatus" default:"enabled"`
 	}
+
+	ExtentAssignmentConfig struct {
+		PublishExtentTargets       []string `name:"publishExtentTargets" default:"/=4"`
```

Comment:
naming nits: Please make it clear that the config values here are numbers, `Targets` are confusing to me. One suggestion would be: 

    NumPublisherExtentsByPath []string
    NumConsumerExtentsByPath []string
    NumRemoteConsumerExtentsByPath []string